### PR TITLE
Updated neighbourhood records with dual hierarchies

### LINKED ISSUE #746: Verify all dual hierarchies for new neighbourhood imports
The following records were recently imported via the neighbourhood update issues. These records have a `wof:parent_id` of `-3`, though many (possibly all) of these records' hierarchies do not include the correct dual hierarchy and instead only have a single hierarchy.

Verify that all these records have the correct hierarchies and `wof:controlled` properties. Rebuild hierarchies in a separate PR that will not require PIP work.

```
Alexandria (X)
1108724979
420547827

Amsterdam (X, partial)
1108802535
1108802955
1108803205
1108803313
1108803333
1108803391
1108803439
1108803607
1108803625
1108803783
1108803809
1108803829
1108803923
1108803945
1108803963
1108803991
1108804017
1108804037
1108804047
1108804087
1108804109
1108804175
1108804229
1108804247
1108804289

Arlington (X)
1108726423

Boston (X)
1108705213
1108711419
1108711421

Cambridge (X)
1108712253
1108713379

Chicago (X)
1024497031
975150323
991907383

DC (X)
85805519
85832623
420510953
1108723549
1108723849
1108723981
1108724017
1108724035
1108724051
1108724053
1108724057
1108723847
1108726811

Helsinki (X, controlled props only)
85907971
1108726995
1108727017
1108727085
1108727093
1108727095
1108727101
1108727169

LA (X)
1108561749
1108561751
1108691447

Minneapolis (X)
1108800221
1108800005
1108800007
1108800013
1108800089
1108800091
1108800097

NYC (X)
907189669
907217995

Philadelphia (X)
1108721785
1108721787
1108721789
1108721887
1108721901
1108722077
1108723359
1108723361
1108723363
1108723379
1108723409
85893367

Portland (X)
1108713505
1108713507
1108714053
1108715777

Seattle (X)
890536749
907129183

The Hague (X)
1108808031
```

### PR DESCRIPTION
**Pull request by stepps00**
Fixes #746.

This PR updated the `wof:hierarchy`, `wof:parent_id`, and `wof:controlled` properties for records in recent neighbourhood update markets that had "funky" hierarchies. Each commit takes care of a single market.

This work basically rebuilds hierarchies of records by hand, so no additional PIP would should be required. All controlled props and parent_ids were updated to make sure future PIP work will not overwrite these changes.

### CODE REVIEW

File: data/110/856/174/9/1108561749.geojson [hunk 1]
```diff
@@ -27,16 +27,49 @@
...
-    "wof:belongsto":[],
+    "wof:belongsto":[
+        "85886323",
+        102191575,
+        1108692797,
+        85633793,
+        85923517,
+        102086957,
+        85688637,
+        "420551221"
```
Comment (nvkelso) [inline data/110/856/174/9/1108561749.geojson]: Beware "quoted" int values in `wof:belongsto`.

File: data/110/856/174/9/1108561749.geojson [hunk 2]
```diff
@@ -27,16 +27,49 @@
...
+    "wof:hierarchy":[
+        {
+            "continent_id":102191575,
+            "country_id":85633793,
+            "county_id":102086957,
+            "locality_id":85923517,
+            "macrohood_id":1108692797,
+            "microhood_id":1108561749,
+            "neighbourhood_id":"85886323",
+            "region_id":85688637
```
Comment (nvkelso) [inline data/110/856/174/9/1108561749.geojson]: The source of your "quoted" problems?

File: data/110/856/174/9/1108561749.geojson [hunk 3]
```diff
@@ -27,16 +27,49 @@
...
+    "wof:geomhash":"b4bf95e668be14321cddd342236e9b5f",
+    "wof:hierarchy":[
+        {
+            "continent_id":102191575,
+            "country_id":85633793,
+            "county_id":102086957,
+            "locality_id":85923517,
+            "macrohood_id":1108692797,
+            "microhood_id":1108561749,
+            "neighbourhood_id":"85886323",
```
Comment (nvkelso) [inline data/110/856/174/9/1108561749.geojson]: The source of your "quoted" problems? (and again in next hierarchy entry)

File: data/110/856/175/1/1108561751.geojson [hunk 4]
```diff
@@ -31,16 +31,49 @@
...
+    "wof:geomhash":"9643db0904766617198202e2db6e099b",
+    "wof:hierarchy":[
+        {
+            "continent_id":102191575,
+            "country_id":85633793,
+            "county_id":102086957,
+            "locality_id":85923517,
+            "macrohood_id":1108692797,
+            "microhood_id":1108561751,
+            "neighbourhood_id":"85855873",
```
Comment (nvkelso) [inline data/110/856/175/1/1108561751.geojson]: Ditto "quotted"

**Comment by nvkelso** [comment]
I spot checked a few of the hierarchies and they look good – except for the "quoted" id numbers. Fix that and then pass to Aaron.

**Comment by stepps00** [comment]
No action on the above comments - the string values in hierarchies are being passed down from parent records. 

Issue https://github.com/whosonfirst-data/whosonfirst-data/issues/450 was opened and will track a bulk property type update for the `wof:belongsto` and `wof:hierarchy` properties.

**Comment by stepps00** [comment]
Related: https://github.com/whosonfirst-data/whosonfirst-data/compare/master...pr-651

**Comment by nvkelso** [comment]
PIP is required.

**Pull request merged**